fastchat/conversation.py

```diff
@@ -530,7 +540,7 @@ def get_conv_template(name: str) -> Conversation:
 # Deepseek code default template
 register_conv_template(
     Conversation(
-        name="deepseek",
+        name="deepseek-coder",
         system_template="You are an AI programming assistant, utilizing the DeepSeek Coder model, developed by DeepSeek Company, and you only answer questions related to computer science. For politically sensitive questions, security and privacy issues, and other non-computer science questions, you will refuse to answer.",
         roles=("### Instruction:", "### Response:"),
         sep="\n",
```

Comment:
so deepseek-coder's sep_style is different from deepseek?
```
        sep_style=SeparatorStyle.DEEPSEEK_CHAT,
```

Reply:
Yup, it's style is different according to this: https://huggingface.co/deepseek-ai/deepseek-coder-33b-instruct/blob/main/tokenizer_config.json

Deepseek-coder, seems to do 
```
'###Instruction:\n' + message['content'] + '\n'
'###Response:\n + message['content'] + '\n<|EOT|>\n'
```

So, there's an extra `'\n'` after instruction or response.